x-pack/plugins/uptime/public/components/monitor/ping_list/ping_list_header.tsx

```diff
@@ -27,7 +27,7 @@ export const PingListHeader = () => {
       <EuiFlexItem grow={false}>
         <StatusFilter />
       </EuiFlexItem>
-      <EuiFlexItem grow={false}>
+      <EuiFlexItem grow={false} style={{ maxWidth: '135px' }}>
```

Comment:
Simple patch to prevent locations filter dropdown from getting super wide when many locations are selected.